keyboards/mothwing/rules.mk

```diff
@@ -0,0 +1 @@
+SPLIT_KEYBOARD = yes
```

Comment:
this seems to be required when split. is it right?

Reply:
```suggestion
```